dmoj/problem.py

```diff
@@ -99,7 +99,9 @@ def __del__(self):
 
 class BatchedTestCase(object):
     def __init__(self, batch_no, config, problem):
-        self.config = config
+        self.config = ConfigNode(config.raw_config, defaults={
```

Comment:
What?? What about the things inherited from the global level??? Why don't you move this to the global config?